styles/home.css

```diff
@@ -18,8 +18,11 @@ h3, section#banner h1 {
 }
 
 h4 {
-  font-size: 1.375rem;
+  font-size: 1.125rem;
   line-height: 1.272em;
+  margin-bottom: 0.5rem;
+  color: #14142b;
+  font-weight: 700;
```

Comment:
please add h4 styles for diffrent screen size

btw good job.

Reply:
made the changes as per your suggestions.